ARrehab/ARrehab/Player.swift

```diff
@@ -28,21 +28,21 @@ class Player : TileCollider, HasModel, HasAnchoring {
     
     override func onCollisionBegan(tile: Tile) {
         self.onTile = tile
-        //self.onTile.changeMaterials(materials: [SimpleMaterial(color: .blue, isMetallic: false)])
+        print("CURRENTLY ON:",onTile?.tileName)
+        self.onTile!.changeMaterials(materials: [SimpleMaterial(color: .blue, isMetallic: false)])
         //super.onCollisionBegan(tile: tile)
     }
     
     override func onCollisionEnded(tile: Tile) {
-        /*if tile != self.onTile {
-            tile.changeMaterials(materials: [SimpleMaterial(color: .clear, isMetallic: false)])
-        }*/
+        //print("CURRENTLY ON:",onTile?.tileName)
+        tile.changeMaterials(materials: [SimpleMaterial(color: .clear, isMetallic: false)])
         //super.onCollisionEnded(tile: tile)
     }
 }
 
 class TileCollider : Entity, HasCollision {
     
-    static let defaultCollisionComp = CollisionComponent(shapes: [ShapeResource.generateBox(width: 0.0, height: 0.1, depth: 0.0)], mode: .trigger, filter: CollisionFilter(group: .default, mask: Tile.TILE_COLLISION_GROUP))
+    static let defaultCollisionComp = CollisionComponent(shapes: [ShapeResource.generateBox(width: 0.0, height: 0.1 / Tile.SCALE, depth: 0.0)], mode: .trigger, filter: CollisionFilter(group: .default, mask: Tile.TILE_COLLISION_GROUP))
```

Comment:
I think the TileCollider is meant to be a single point in space. The tiles themselves are the ones with a larger collision box.
1. does the tile collider work with a CollisionComponent that has 0 width and 0 depth?
2. Confirm that the TileCollider is not supposed to be a single point in space or change it back to being just a single point in space.

Reply:
See #22.